uploader/wiki_uploader.py

```diff
@@ -43,3 +43,28 @@ def upload_file(self, file_name, file_stream, description=""):
             return False, {}
         else:
             return True, upload_result["imageinfo"]
+
+
+def get_initial_page_text(
+    license="", summary="", category="", date_of_creation="", source="", author=""
+):
+    description = "|description=" + summary
+    date_of_creation = "|date=" + date_of_creation
+    source = "|source=" + source
+    author = "|author=" + author
+    return """=={{{{int:filedesc}}}}==
+{{{{Information
+ {0}
+ {1}
+ {2}
+ {3}
+}}}}
+
+
+=={{{{int:license-header}}}}==
+{{{{{4}}}}}
+
+[[Category:{5}]] 
```

Comment:
we should add categories, only if they exist :)

Reply:
Noted!